flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/expressions/validation/ArrayTypeValidationTest.scala

```diff
@@ -26,11 +26,6 @@ import org.junit.Test
 
 class ArrayTypeValidationTest extends ArrayTypeTestBase {
 
-  @Test(expected = classOf[ValidationException])
-  def testImplicitTypeCastTableApi(): Unit = {
```

Comment:
Can we add a test to `org.apache.flink.table.planner.expressions.ArrayTypeTest` for this if it works now.